Tools/README

```diff
@@ -26,6 +28,12 @@ i18n            Tools for internationalization. pygettext.py
 
 importbench     A set of micro-benchmarks for various import scenarios.
 
+inspection      Tooling for PEP-678 "Safe external debugger interface for CPython".
+
+jit             Tooling for building the JIT.
+
+lockbench       Benchmarks for PyMutex and critical sections.
```

Comment:
Likewise?

Reply:
Nope, because these are micro-benchmarks for very specific use-cases, basically not user-facing ones, but ones used for development.

We have many existing ones already. From time to time they are getting removed.
But, anyway I am just describing what exists right now, we can start a new discussion about removing some things from there.